src/CharClass.ts

```diff
@@ -0,0 +1,34 @@
+import {Skill} from './Skill'
+import {Stat} from './Stat'
+import {Discipline} from './Discipline'
+import {Power} from './Power'
+
+const classes = ['Assassin', 'Barbarian', 'Crusader'] // Temporary list just to test setClass method.
+
+interface ICharClass {
```

Comment:
I would match what the game does and call this `Profession`, `Class` (besides being a bad name) refers to `Fighter`, `Mage`, etc.